### ISSUE TITLE
How does DIGITS load segmentation label images?

### ISSUE THREAD

**Issue by Oceanland-428** [NVIDIA/DIGITS#2094]
Hi!

I'm doing training on a custom network for segmentation using Tensorflow. The loss is strange. When I checked the self.y values, they are not the same as I expected. I've built the LMDB file such thatall the label images should have labels in range [0, 5], but the pixel value I got for self.y is out of that range (eg. 7.26). I've chosen 'NONE' for 'subtract mean'. 

I value is clearly no the normalized value (which should in range of 0 to 1). I'd like to know how does DIGITS load the label image? Does it normalize the image and do something else? Or doe perform any other transformation on the images?

Thank you very much!
